dash_core_components/metadata.json

```diff
@@ -139,73 +139,6 @@
       }
     }
   },
-  "src/components/Confirm.react.js": {
```

Comment:
Curious why this chunk of code was removed?

Reply:
That is curious... I have no idea why it's there in the first place, as there's no Confirm component or anything. Could it be related to the new Confirmation modal?